[2023-04-09T00:33:17.227+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:28:13.687031+00:00 [queued]>
[2023-04-09T00:33:17.234+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:28:13.687031+00:00 [queued]>
[2023-04-09T00:33:17.235+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:33:17.235+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-04-09T00:33:17.236+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:33:17.244+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-09 00:28:13.687031+00:00
[2023-04-09T00:33:17.250+0000] {standard_task_runner.py:55} INFO - Started process 9703 to run task
[2023-04-09T00:33:17.254+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'is_api_available', 'manual__2023-04-09T00:28:13.687031+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmp1xyj0g60']
[2023-04-09T00:33:17.256+0000] {standard_task_runner.py:83} INFO - Job 63: Subtask is_api_available
[2023-04-09T00:33:17.317+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:28:13.687031+00:00 [running]> on host c12a518d534c
[2023-04-09T00:33:17.368+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T00:28:13.687031+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T00:28:13.687031+00:00
[2023-04-09T00:33:17.373+0000] {http.py:120} INFO - Poking: api/
[2023-04-09T00:33:17.383+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-09T00:33:17.386+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-09T00:33:17.627+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-09T00:33:17.645+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=is_api_available, execution_date=20230409T002813, start_date=20230409T003317, end_date=20230409T003317
[2023-04-09T00:33:17.678+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-09T00:33:17.703+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
